gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
```

Comment:
Agreed.  Admittedly this was a copy and paste procedure from the description of the related talk we'll delivering at OSCON on the InnerSource marketplace being created at Fidelity.  Time to clean it up and have it speak more succinctly to the Gig Marketplace alone as a discrete pattern.